installer/common/install-operators.sh

```diff
@@ -73,6 +73,7 @@ fi
 
 # Spark Operator
 if [ "$installSparkOperator" = true ]; then
+    SPARK_CHART_VERSION=0.4.0
```

Comment:
This version is already available in shared.sh as `sparkOperatorChartVersion`. The only one not listed in shared.sh is the NFS chart version. Might be good to keep them all in 1 place?